nodejs/week1/avg.js

```diff
@@ -0,0 +1,8 @@
+const arg = process.argv.slice(2);
+let number = 0;
```

Comment:
Always a good idea to name variables that represent the subject. Like total or sum?